### PR TITLE
Seeds data for simple example

### PR DESCRIPTION
**Pull request by Tica2**
Solve 
https://github.com/elixir-lang/ecto/issues/945

### CODE REVIEW

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

File: examples/simple/README.md [hunk 1]
```diff
@@ -8,8 +8,24 @@ Then, from the command line:
 * `mix ecto.create`
 * `mix ecto.migrate`
 * `iex -S mix`
+* `mix run priv/repo/seeds.exs`
```
Comment (ericmj) [inline examples/simple/README.md]: This should be run before `iex -S mix`.

Comment (Tica2) [inline examples/simple/README.md]: Sure :D I will come with fix. Thanks!

Comment (Tica2) [inline examples/simple/README.md]: Fix in https://github.com/elixir-lang/ecto/pull/957

**Pull request merged**